x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/DataFrameTransformPersistentTasksExecutor.java

```diff
@@ -131,7 +131,7 @@ protected void nodeOperation(AllocatedPersistentTask task, @Nullable DataFrameTr
                     .setInitialPosition(stateAndStats.getTransformState().getPosition())
                     .setProgress(stateAndStats.getTransformState().getProgress())
                     .setIndexerState(currentIndexerState(stateAndStats.getTransformState()));
-                logger.info("[{}] Loading existing state: [{}], position [{}]",
+                logger.debug("[{}] Loading existing state: [{}], position [{}]",
```

Comment:
Double-checking: is this change intended in this PR?

Reply:
@przemekwitek yeah, this change was skipped in a previous PR and I just put it in this one. Low impact and did not want to have to kick of an entire CI run on my previous PR because of a logging level change.

Reply:
No problem, just wanted to make sure that it is not a leftover from your local debugging.